import-scripts/study_directory_uses_updated_filenames.py

```diff
@@ -0,0 +1,538 @@
+        "meta_RNA_Seq_v2_mRNA_median_all_sample_ref_normal_Zscores.txt",
+        "meta_RNA_Seq_v2_mRNA_median_normals.txt",
+        "meta_armlevel_CNA.txt",
+        "meta_bcr_clinical.txt",
+        "meta_bcr_clinical_other.txt",
+        "meta_clinical.txt",
+        "meta_clinical_caises.txt",
+        "meta_clinical_patient.txt",
+        "meta_clinical_sample.txt",
+        "meta_cna_hg18_seg.txt",
+        "meta_cna_hg19_seg.txt",
+        "meta_drug_treatment_AUC.txt",
+        "meta_drug_treatment_IC50.txt",
+        "meta_drug_treatment_ZSCORE.txt",
+        "meta_expression.txt",
+        "meta_expression_Zscores.txt",
+        "meta_expression_all_sample_Zscores.txt",
+        "meta_expression_median.txt",
+        "meta_expression_merged.txt",
+        "meta_expression_merged_Zscores.txt",
+        "meta_expression_merged_median_Zscores.txt",
+        "meta_expression_miRNA.txt",
+        "meta_expression_other_Zscores.txt",
+        "meta_fusions.txt",
+        "meta_fusions_gml.txt",
+        "meta_gene_matrix.txt",
+        "meta_gistic_genes_amp.txt",
+        "meta_gistic_genes_del.txt",
+        "meta_gsva_pvalues.txt",
+        "meta_gsva_scores.txt",
+        "meta_linear_CNA.txt",
+        "meta_log2CNA.txt",
+        "meta_mRNA_median_Zscores.txt",
+        "meta_mRNA_median_all_sample_Zscores.txt",
+        "meta_mRNA_outliers.txt",
+        "meta_mRNA_seq_fpkm.txt",
+        "meta_mRNA_seq_fpkm_capture.txt",
+        "meta_mRNA_seq_fpkm_capture_Zscores.txt",
+        "meta_mRNA_seq_fpkm_capture_all_sample_Zscores.txt",
+        "meta_mRNA_seq_fpkm_polya.txt",
+        "meta_mRNA_seq_fpkm_polya_Zscores.txt",
+        "meta_mRNA_seq_fpkm_polya_all_sample_Zscores.txt",
+        "meta_methylation_binary.txt",
+        "meta_methylation_genebodies_hmEPIC.txt",
+        "meta_methylation_genebodies_wgbs.txt",
+        "meta_methylation_hm27.txt",
+        "meta_methylation_hm450.txt",
+        "meta_methylation_promoters_hmEPIC.txt",
+        "meta_methylation_promoters_wgbs.txt",
+        "meta_miRNA_median_Zscores.txt",
+        "meta_microbiome.txt",
+        "meta_mrnaseq_fcount.txt",
+        "meta_mrnaseq_rsem.txt",
+        "meta_mutational_signature_contribution_v2.txt",
+        "meta_mutational_signature_contribution_v3.txt",
+        "meta_mutational_signature_pvalue_v2.txt",
+        "meta_mutational_signature_pvalue_v3.txt",
+        "meta_mutations_extended.txt",
+        "meta_mutations_uncalled.txt",
+        "meta_mutsig.txt",
+        "meta_phosphoprotein_quantification.txt",
+        "meta_protein_quantification.txt",
+        "meta_protein_quantification_Zscores.txt",
+        "meta_protein_quantification_zscores.txt",
+        "meta_resource_definition.txt",
+        "meta_resource_sample.txt",
+        "meta_resource_study.txt",
+        "meta_rppa.txt",
+        "meta_rppa_Zscores.txt",
+        "meta_timeline.txt"
+        ]
+    filenames_in_use_before_update_set = set()
```

Comment:
maybe keep constants as a list of strings then use the replace later in a separate function in its own routine

Reply:
done